src/renderer/components/history/Table.vue

```diff
@@ -192,8 +194,9 @@ export default {
   }
 
   .body {
-    height: 90%;
+    flex-grow: 1;
     overflow-y: auto;
+    min-height: 0; /* for Firefox */
```

Comment:
We don't need anything for Firefox. Electron uses Chrome.

Reply:
I know, its just that if we ever used this code in firefox, we would need that and would probably never know to do it.